src/redux/auth/actionTypes.ts

```diff
@@ -0,0 +1,6 @@
+export enum AuthActionTypes {
+    SET_TOKENS = 'SET_TOKENS',
+    FETCH_START = 'FETCH_START',
+    FETCH_END = 'FETCH_END',
+    FETCH_ERROR = 'FETCH_ERROR'
+}
```

Comment:
you have non-unique constants here
at least consider the AUTH_SET_TOKENS, AUTH_FETCH_START etc 
or seek for the best practices

Reply:
p.s. you can keep the enum name as is, but the value should be fixed